maud_macros/tests/warnings/warn_on_keyword_names.stderr

```diff
@@ -0,0 +1,6 @@
+warning: found keyword `if` - should this be a `@if`?
+ --> $DIR/warn_on_keyword_names.rs:9:9
+  |
+9 |         if {} //~WARNING found keyword `if` - should this be a `@if`?
```

Comment:
Apparently not. I can't get the comment to work, inside or outside the macro: this might be something in `compiletest`, because they have no UI tests in their examples to demonstrate how it should work - and I can't see where else I might be going wrong.

I'll delete the comment and just keep the error file for now.